### DISCUSSION TITLE
transplant

### DISCUSSION THREAD

**Discussion by NCreak** [eProsima/Fast-DDS#3708] [Q&A]
I want to ask how to transplant fastdds to android

**Comment by JesusPoderoso** [comment]
Hi @NCreak, what do you mean with transplant Fast DDS to Android?
If you mean crosscompilation, it would be as [crosscompiling any CMake project](https://developer.android.com/ndk/guides/cmake).

**Reply by NCreak** [reply]
Hi, I build out library using NDK, but the namespace of the library compiled by NDK is std::__ndk1, but I use the namespace of std::__1:: in the Android system, which will cause undefined errors
